diff -Nur a/arch/arm/boot/dts/sun8i-h3-orangepi-2.dts b/arch/arm/boot/dts/sun8i-h3-orangepi-2.dts
--- a/arch/arm/boot/dts/sun8i-h3-orangepi-2.dts	2016-03-27 16:16:52.000000000 +0200
+++ b/arch/arm/boot/dts/sun8i-h3-orangepi-2.dts	2016-03-31 21:50:18.295857936 +0200
@@ -184,3 +184,12 @@
 	usb1_vbus-supply = <&reg_usb1_vbus>;
 	status = "okay";
 };
+
+&emac {
+	phy = <&phy1>;
+	phy-mode = "mii";
+	status = "okay";
+	phy1: ethernet-phy@1 {
+		reg = <0>;
+	};
+};
\ Brez nove vrstice ob koncu datoteke
diff -Nur a/arch/arm/boot/dts/sun8i-h3-orangepi-pc.dts b/arch/arm/boot/dts/sun8i-h3-orangepi-pc.dts
--- a/arch/arm/boot/dts/sun8i-h3-orangepi-pc.dts	2016-03-27 16:16:52.000000000 +0200
+++ b/arch/arm/boot/dts/sun8i-h3-orangepi-pc.dts	2016-03-31 21:49:37.194689646 +0200
@@ -165,3 +165,12 @@
 	/* USB VBUS is always on */
 	status = "okay";
 };
+
+&emac {
+	phy = <&phy1>;
+	phy-mode = "mii";
+	status = "okay";
+	phy1: ethernet-phy@1 {
+		reg = <0>;
+	};
+};
\ Brez nove vrstice ob koncu datoteke
diff -Nur a/arch/arm/boot/dts/sun8i-h3-orangepi-plus.dts b/arch/arm/boot/dts/sun8i-h3-orangepi-plus.dts
--- a/arch/arm/boot/dts/sun8i-h3-orangepi-plus.dts	2016-03-27 16:16:52.000000000 +0200
+++ b/arch/arm/boot/dts/sun8i-h3-orangepi-plus.dts	2016-03-31 23:49:50.104870880 +0200
@@ -93,3 +93,14 @@
 &usbphy {
 	usb3_vbus-supply = <&reg_usb3_vbus>;
 };
+
+&emac {
+        pinctrl-names = "default";
+        pinctrl-0 = <&emac_rgmii_pins>;
+        phy = <&phy1>;
+        phy-mode = "rgmii";
+        status = "okay";
+        phy1: ethernet-phy@1 {
+                  reg = <0>;
+        };
+};
\ Brez nove vrstice ob koncu datoteke
